lib/data/landing_page_content_items/landing_page.yaml

```diff
@@ -77,6 +77,22 @@ blocks:
     content: <p>Right content!</p>
 - type: govspeak
   content: <h2>Statistics</h2>
+- type: hero
+  mid_page: true
```

Comment:
I think `theme: middle_left` might be better.

Couple of reasons for that:

1. mid_page makes it sounds like somethings going to be in the middle of the _page_, which isn't quite what this is. The content is in the middle of the hero, not the page.
2. Using booleans for this kind of setting leads to a common problem with proliferation of options. At some point, maybe we'll want one of these with the content on the right, and we'll add `right_page: true`, and then we can do bottom-left, mid-left, bottom-right, and mid-right. Then, later, someone will want to do `top`, and the code will have to decide how to handle invalid combinations like `mid_page: true; top_page: true`
3. Other blocks use `theme` (e.g. the two column layout has a `two_thirds_one_third` "theme")

Reply:
Thanks, yep theme is much better and consistent with other blocks. Updated now.